app/src/ui/multi-commit-operation/choose-branch/rebase-choose-branch-dialog.tsx

```diff
@@ -89,15 +89,15 @@ export class RebaseChooseBranchDialog extends React.Component<
       currentBranch !== null &&
       selectedBranch.name === currentBranch.name
 
-    const areCommitsToRebase =
+    const areCommitsToBringFromOtherBranch =
```

Comment:
I think this would be a better name:
```suggestion
    const areCommitsToRebaseOnto =
```